aten/src/ATen/native/native_functions.yaml

```diff
@@ -2483,6 +2524,17 @@
 
 - func: norm.out(Tensor self, Scalar? p, int[1] dim, bool keepdim=False, *, Tensor(a!) out) -> Tensor(a!)
 
+- func: norm.names_ScalarOpt_dim_dtype(Tensor self, Scalar? p, Dimname[1] dim, bool keepdim, *, ScalarType dtype) -> Tensor
+  variants: function, method
+
+- func: norm.names_ScalarOpt_dim(Tensor self, Scalar? p, Dimname[1] dim, bool keepdim=False) -> Tensor
```

Comment:
I'm a bit unfamiliar with the dot notation being used now (I think it's new?), .. but seems like most suffices are lower case. Does ScalarOpt have to be camelcase here? it looks a bit out of place and not sure if there's any significance to that or if it has to match something else.

Reply:
It doesn't actually matter what the overload names are. I'm making it so that the overload names match the non-named versions (see norm.ScalarOpt_dim).